File: www/VoiceRecognize.js
var exec = require('cordova/exec');

function VoiceRecognize() {}

VoiceRecognize.prototype.recognizeVoiceModel = function( success, error) {
    exec(success, error, "VoiceRecognize", "recognizeVoiceModel", []);
}

VoiceRecognize.prototype.trainVoiceModel = function( success, error) {
    exec(success, error, "VoiceRecognize", "trainVoiceModel", []);
}

VoiceRecognize.prototype.deleteVoiceModel = function( success, error) {
    exec(success, error, "VoiceRecognize", "deleteVoiceModel", []);
}

module.exports = new VoiceRecognize();

File: src/ios/VoiceRecognize.m
//
//  VoiceRecognize.m
//  SuperApp_Ionic
//
//  Created by gaoqh on 2017/3/30.
//
//

#import "VoiceRecognize.h"
#import "Reachability.h"
#import "iflyMSC/iFlyISVRecognizer.h"
#import "iflyMSC/IFlyISVDelegate.h"

@implementation VoiceRecognize

- (void)recognizeVoiceModel:(CDVInvokedUrlCommand*)command
{
//    CDVPluginResult* pluginResult = nil;
//    NSString* echo = [command.arguments objectAtIndex:0];
//
//    if (echo != nil && [echo length] > 0) {
//        pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:echo];
//    } else {
//        pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR];
//    }
//
//    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
}

- (void)trainVoiceModel:(CDVInvokedUrlCommand*)command
{
    
}

- (void)deleteVoiceModel:(CDVInvokedUrlCommand*)command
{
    
}


@end
